### ISSUE TITLE
Hashdard should use jsx double quotes

### ISSUE THREAD

**Issue by duranmla** [hashlabs/hashdard#4]
The base eslint has jsx double quotes by default, but the standard-jsx doesn't

![hashdard_eslintrc_json_at_master_ _hashlabs_hashdard](https://cloud.githubusercontent.com/assets/1425162/23077277/4fc34b72-f51a-11e6-8ede-a3e910ab2e95.jpg)

We need to overwrite that rule of standard-jsx to make it work as expected

![eslint-config-standard-jsx_eslintrc_json_at_1d24e6cfb18b58fc1300a48b18b16873b3697692_ _feross_eslint-config-standard-jsx](https://cloud.githubusercontent.com/assets/1425162/23077316/73872e16-f51a-11e6-8632-e3ecec8ccabe.jpg)

**Comment by carlows** [comment]
@duranmla can you explain what issue are you having here?

That rule is being overwritten here: https://github.com/hashlabs/eslint-config-hashdard/blob/master/eslintrc.json#L17

If I remember correctly that PR fixed this issue, isn't it working for you? Did you update hashdard to version `1.0.3`?

**Comment by duranmla** [comment]
Yes @Carlows is the only rule I am having troubles with. It happens with the cli command and as expected with the integration with atom.

![1__hashlabs_alexis____hashlabs_internal_simian-ci_client_src_js__zsh_](https://cloud.githubusercontent.com/assets/1425162/23136552/aa063a76-f773-11e6-9dd2-2601d1d17349.jpg)

**Comment by duranmla** [comment]
@Carlows @orlando this is working for you guys?

**Comment by orlando** [comment]
@duranmla no idea, I haven't written JavaScript lately

**Comment by carlows** [comment]
@duranmla I just bootstrapped an app with `create-react-app` and for a component like this:

```
import React, { Component } from 'react';
import logo from './logo.svg';
import './App.css';

class App extends Component {
  render() {
    return (
      <div className='App'>
        <div className="App-header">
          <img src={ logo } className="App-logo" alt="logo" />
          <h2>Welcome to React</h2>
        </div>
        <p className="App-intro">
          To get started, edit <code>src/App.js</code> and save to reload.
        </p>
      </div>
    );
  }
}

export default App;
```

This is what I get from hashdard:

```
/Users/hashlabs/Documents/CarlosProjects/test-react/src/App.js:8:22: Unexpected usage of singlequote.
```

So yeah, this is working for me, it expects double quotes on JSX tags.

My versions are:

- hashdard v1.0.4
- npm v4.4.1

And I'm running hashdard from the cli.
